### PR TITLE
Mvj 177 footer links

### PR DESCRIPTION
**Pull request by henrinie-nc**

### CODE REVIEW

File: src/i18n/sv/common.json [hunk 1]
```diff
@@ -355,7 +352,7 @@
     },
     "registerStatement": {
       "label": "Registerbeskrivning",
-      "url": "#"
+      "url": "https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Rekisteriselosteet/Kymp/Tonttien%20hankinnan%20ja%20luovutuksen%20asiakasrekisteri.pdf"
```
Comment (juho-kettunen-nc) [inline src/i18n/sv/common.json]: I think the swedish version link was posted to ticket a few minutes ago, so we can use that

File: src/i18n/sv/common.json [hunk 2]
```diff
@@ -337,12 +337,9 @@
...
-      "facebook": "Helsingfors stad på Facebook",
-      "twitter": "Helsingfors stad på X",
-      "instagram": "Helsingfors stad på Instagram",
-      "linkedin": "Helsingfors stad på LinkedIn"
+    "customerService": {
+      "label": "Stadsmiljösektorns kundservice",
+      "url": "https://www.hel.fi/sv/stadsmiljo-och-trafik/stadsmiljosektorns-kundtjanst"
     },
     "moveToTop": "Gå upp",
     "accessabilityStatement": {
```
Comment (juho-kettunen-nc) [inline src/i18n/sv/common.json]: This isn't broken or part of the changes, but this variable name has a typo. Should be `accessibilityStatement`.

**Review by juho-kettunen-nc — CHANGES_REQUESTED** [review]
Use the swedish register statement URL, otherwise good.

File: src/i18n/sv/common.json (hunk 1 above)
Comment (henrinie-nc) [reply src/i18n/sv/common.json]: added

File: src/i18n/sv/common.json (hunk 2 above)
Comment (henrinie-nc) [reply src/i18n/sv/common.json]: @juho-kettunen-nc fixed, could yo ure-review?

**Review by juho-kettunen-nc — APPROVED** [review]

**Pull request merged**